README.md

```diff
@@ -133,16 +133,35 @@ Ubuntu 16.04     | no
 Ubuntu 18.04     | yes
 
 
-## Localization
+## Localization and translation
+
+Zonemaster comes with translation for the following
+languages
+
+* English (en, en_US.UTF-8)
+* French (fr, fr_FR.UTF-8)
+* Swedish (sv, sv_SE.UTF-8)
+* Danish (da, da_DK.UTF-8)
+
+The support of localization or translation is however not complete due to
```

Comment:
IMHO this part should go into the "Notable bugs and issues" section.

Reply:
What part?

Reply:
Line 146 and below.

Reply:
Updated.